src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -123,19 +123,20 @@
 
   For background, see
   http://blog.colinwilliams.name/blog/2015/01/26/alternative-clojure-dot-test-integration-with-test-dot-check/"
-    :arglists '([name bindings body] [name num-tests-or-options bindings body])}
+   :arglists '([name bindings & body] [name num-tests-or-options bindings & body])}
   [name & check-decl]
-  (let [[num-tests-or-options bindings body] (cond
-                                               (and (or number? (first check-decl)
-                                                        map? (first check-decl))
-                                                    (vector? (second check-decl)))
-                                               [(first check-decl) (second check-decl) (nnext check-decl)]
-
-                                               (vector? (first check-decl))
-                                               [nil (first check-decl) (next check-decl)]
-
-                                               :else (throw #?(:clj (IllegalArgumentException. "Arguments to `checking` must be either [name bindings body] or [name num-tests-or-options bindings body]")
-                                                               :cljs (js/Error. "Arguments to `checking` must be either [name bindings body] or [name num-tests-or-options bindings body]"))))
+  (let [[num-tests-or-options bindings body]
+        (cond
+          (and (or (number? (first check-decl))
+                   (map? (first check-decl)))
+               (vector? (second check-decl)))
+          [(first check-decl) (second check-decl) (nnext check-decl)]
+
+          (vector? (first check-decl))
+          [nil (first check-decl) (next check-decl)]
+
+          :else (throw #?(:clj  (IllegalArgumentException. "Arguments to `checking` must be either [name bindings body] or [name num-tests-or-options bindings body]")
+                          :cljs (js/Error. "Arguments to `checking` must be either [name bindings body] or [name num-tests-or-options bindings body]"))))
```

Comment:
you can avoid the duplication here by pushing the `#?` down to just the `IllegalArgumentException.` and `js/Error.`